packages/design-system/src/interfaces/forms.d.ts

```diff
@@ -1,5 +1,9 @@
 // <reference types="react" />
 
+export type SearchBarSize = "basic" | "compact";
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+type OmitSearchBar = Omit<ISearchBarProps, "debounceInMs">;
```

Comment:
Something bad happened here `OmitSearchBar` type is based on `ISearchBarProps` but `ISearchBarProps` extends `OmitSearchBar`. It's like a circular dependency. Also I don't know why you need to omit `debounceInMs`?